capture/src/event.rs

```diff
@@ -51,14 +65,14 @@ enum RawRequest {
     /// Batch of events
     Batch(Vec<RawEvent>),
     /// Single event
-    One(RawEvent),
+    One(Box<RawEvent>),
 }
 
 impl RawRequest {
     pub fn events(self) -> Vec<RawEvent> {
         match self {
             RawRequest::Batch(events) => events,
-            RawRequest::One(event) => vec![event],
+            RawRequest::One(event) => vec![*event],
```

Comment:
I can't find a way of avoiding this without `unsafe`, and unless profiling tells us copying is a huge issue let's not

Reply:
`RawRequest::Batch` is the happy path for posthog-js anyway, the other one is for compat so it's not a huge deal if it's slower